ext/iconv/php_iconv.h

```diff
@@ -65,14 +65,6 @@ PHP_FUNCTION(iconv_mime_encode);
 PHP_FUNCTION(iconv_mime_decode);
 PHP_FUNCTION(iconv_mime_decode_headers);
 
-ZEND_BEGIN_MODULE_GLOBALS(iconv)
-	char *input_encoding;
-	char *internal_encoding;
-	char *output_encoding;
-ZEND_END_MODULE_GLOBALS(iconv)
-
-#define ICONVG(v) ZEND_MODULE_GLOBALS_ACCESSOR(iconv, v)
-
 #if defined(ZTS) && defined(COMPILE_DL_ICONV)
 ZEND_TSRMLS_CACHE_EXTERN()
 #endif
```

Comment:
I think these 3 lines can be removed now.

Reply:
Dropped them, will see if everything builds 🤞

Reply:
Works fine. :)